### ISSUE TITLE
[3dEXT] error: default member initializer for 'w' needed

### ISSUE THREAD

**Issue by ajh123** [OneLoneCoder/olcPixelGameEngine#225]
I have updated the car crime city part 2 to use 3d extension v0.4 and pixel engine v2.15 and i get
```
In file included from main.cpp:63:
In file included from ./cCarCrimeCity.h:65:
./olcPGEX_Graphics3D.h:190:138: error: default member initializer for 'w' needed
      within definition of enclosing class 'GFX3D' outside of member functions
  ...pos, olc::GFX3D::vec3d dir = { 0.0f, 0.0f, 1.0f }, float fParam = 0.0f);
```

However on 3d extension v0.3 it was just on a different line,
If it helps i'm on MacOS.

**Comment by brccabral** [comment]
I think this is solved by #338

**Comment by ajh123** [comment]
This is an old issue now.
Closing now ...
